beets/dbcore/query.py

```diff
@@ -606,6 +637,8 @@ def open_right_endpoint(self):
         """
         precision = self.precision
         date = self.date
+        if 'relative' == self.precision:
+            return date
```

Comment:
Can `self.precision` ever be "relative" in this version? Is this possibly out of date?